proto/regen/ecocredit/v1beta1/state.proto

```diff
@@ -0,0 +1,213 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1beta1;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// ClassInfo represents the high-level on-chain information for a credit class.
+message ClassInfo {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "class_id"}
+    };
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
+
+    // admin is the admin of the credit class.
+    string admin = 2;
+
+    // issuers are the approved issuers of the credit class.
+    repeated string issuers = 3;
```

Comment:
In keeping with 1NF (which I think we should), let's move this to its own join table